src/applications/appeals/10182/config/form.js

```diff
@@ -50,6 +51,17 @@ import {
 import manifest from '../manifest.json';
 import hearingType from '../pages/hearingType';
 
+/**
+ * add custom focus for the contact info mailing address edit page
+ * fixes issue [#61243](https://github.com/department-of-veterans-affairs/va.gov-team/issues/61243)
+ */
+const confirmContactInfoEditMailingAddress = {
+  ...contactInfo.confirmContactInfoEditMailingAddress,
+  scrollAndFocusTarget: () => {
+    focusElement('#main h3');
+  },
```

Comment:
Would it be better to add this as another parameter within the [`ContactInfo` component](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms-system/src/js/components/ContactInfo.jsx#L55)? The `scrollAndFocusTarget` callback is ignored until the form config `useCustomScrollAndFocus` is set to `true`

Reply:
Thank you! <3